src/body.js

```diff
@@ -293,7 +293,7 @@ export function clone(instance, highWaterMark) {
  */
 export function extractContentType(body) {
 	// Body is null or undefined
-	if (body == null) {
+	if (body === null) {
```

Comment:
The comment is no longer accurate since this now only checks for `null` (`=== null`) and doesn't include `undefined` (`== null`)

This is present in a number of places...